charts/redpanda/templates/configmap.yaml

```diff
@@ -53,6 +53,37 @@ limitations under the License.
     {{- $users = append $users $user.name -}}
   {{- end -}}
 {{- end -}}
+
+{{- $configurationBooleanProp := list "admin_api_require_auth"
+"aggregate_metrics"
+"auto_create_topics_enabled"
+"cloud_storage_disable_tls"
+"cloud_storage_enable_compacted_topic_reupload"
+"cloud_storage_enable_remote_read"
+"cloud_storage_enable_remote_write"
+"cloud_storage_enable_segment_merging"
+"cloud_storage_enabled"
+"disable_batch_cache"
+"disable_metrics"
+"disable_public_metrics"
+"enable_controller_log_rate_limiting"
+"enable_coproc"
+"enable_idempotence"
+"enable_leader_balancer"
+"enable_metrics_reporter"
+"enable_pid_file"
+"enable_rack_awareness"
+"enable_sasl"
+"enable_transactions"
+"enable_usage"
+"features_auto_enable"
+"kafka_enable_authorization"
+"kafka_enable_partition_reassignment"
+"kafka_qdc_enable"
+"legacy_group_offset_retention_enabled"
+"memory_abort_on_alloc_failure"
+"release_cache_on_segment_roll"
+"storage_strict_data_init" -}}
```

Comment:
The boolean properties was taken from running cluster v23.1.7 by the following command:

```
kubectl exec redpanda-0 -- curl -sk https://127.0.0.1:9644/v1/cluster_config/schema | jq -c  '.properties | to_entries | .[] | select(.value.type | contains("boolean")) | .key'
```

Reply:
Updated as 23.1.8 added new boolean `use_fetch_scheduler_group`